astropy/wcs/docstrings.py

```diff
@@ -824,9 +824,9 @@ def RA_DEC_ORDER(indent=0):
 """
 
 get_cdelt = """
-get_cdelt() -> ``double array[naxis]``
+get_cdelt() -> numpy.ndarray
```

Comment:
Note: Apparently linking was still attempted despite the double-backticks. This is some C-extension thingy, so I am not going to question it... 🤷